client/clip_client/client.py

```diff
@@ -109,33 +109,40 @@ def encode(
     def encode(self, content, **kwargs):
         if isinstance(content, str):
             raise TypeError(
-                f'content must be an Iterable of [str, Document], try `.encode(["{content}"])` instead'
+                f'Content must be an Iterable of [str, Document], try `.encode(["{content}"])` instead'
             )
+        if hasattr(content, '__len__') and len(content) == 0:
+            raise ValueError('Input content must not be empty')
```

Comment:
I recommend that `encode` should also work for empty docarray.

Reply:
ok i will return something empty instead of raising errors